src/Api/Search/ResponseTransformer/AbstractSuggestProductTransformer.php

```
@@ -74,7 +75,7 @@ protected function collect(SuggestGroup $group, SalesChannelContext $context): a
         return $products;
     }
 
-    protected function enrich(SuggestGroup $group, array $products): void
+    protected function enrich(SuggestGroup $group, array $products, Configuration $config, SalesChannelContext $context): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$config'. (Unused Code Rules)

(UnusedFormalParameter)